.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # Every request must be reviewed and accepted by:
 
-*	@jeroen-meijer @felangel @orestesgaolin
\ No newline at end of file
+*	@jeroen-meijer @felangel @renancaraujo @wolfenrain
```

Comment:
@felangel Mr Meijar has no write access anymore?

Reply:
Hmm yeah we can remove him for now

Reply:
Have u checked it yet?

Reply:
We shall keep his name for homage.